spikesorters/hdsort/hdsort.py

```diff
@@ -47,6 +47,26 @@ class HDSortSorter(BaseSorter):
         'loop_mode': 'local_parfor',
     }
 
+    _params_description = {
+        'detect_threshold': "Threshold for spike detection",
+        'detect_sign': "Use -1 (negative) or 1 (positive) depending "
+                       "on the sign of the spikes in the recording",
+        'filter': "Enable or disable filter",
+        'parfor': "If True, the Matlab parfor is used",
+        'freq_min': "High-pass filter cutoff frequency",
+        'freq_max': "Low-pass filter cutoff frequency",
+        'max_el_per_group': "Maximum number of channels per electrode group",
+        'min_el_per_group': "Minimum number of channels per electrode group",
+        'add_if_nearer_than': "Minimum distance to add electrode to an electrode group",
+        'max_distance_within_group': "Maximum distance within an electrode group",
+        'n_pc_dims': "Number of principal components dimensions to perform initial clustering",
+        'chunk_size': "Chunk size in number of frames for template-matching",
+        'loop_mode': "Loop mode: 'loop', 'local_parfor', 'grid' (requires a grid architecture)",
+    }
+
+    sorter_description = """HDSort is a template-matching spike sorter designed for high density micro-electrode arrays. 
+    For more information see https://doi.org/10.1152/jn.00803.2017"""
+
```

Comment:
This is a cool idea.
I think we should use also this generate `_doc_` for functions lanucher.